packages/edge-functions/dev/deno/workers/config.ts

```diff
@@ -1,19 +1,35 @@
+import type { SerializedError } from '../../shared/types.ts'
 import type { ConfigResponseMessage, Message } from './types.ts'
 
 self.onmessage = async (e) => {
   const message = e.data as Message
 
   if (message.type === 'configRequest') {
     const configs: Record<string, object> = {}
+    const errors: Record<string, SerializedError> = {}
     const imports = Object.entries(message.data.functions).map(async ([name, path]) => {
-      const func = await import(path)
+      try {
+        const func = await import(path)
 
-      configs[name] = func.config ?? {}
+        configs[name] = func.config ?? {}
+      } catch (error: unknown) {
+        if (error instanceof Error) {
+          errors[name] = {
+            message: error.message,
+            name: error.name,
+            stack: error.stack,
+          }
```

Comment:
any chance any of these Errors have a `cause` property that we want to not discard?